src/main/kotlin/app/Analytics.kt

```diff
@@ -61,35 +77,35 @@ object Analytics {
     }
 
     fun trackStart() {
-        trackEvent("/start")
+        trackEvent("start")
     }
 
     fun trackAuth() {
-        trackEvent("/auth")
+        trackEvent("auth")
     }
 
     fun trackConfigSetup() {
-        trackEvent("/config/setup")
+        trackEvent("config/setup")
     }
 
     fun trackConfigChanged() {
-        trackEvent("/config/changed")
+        trackEvent("config/changed")
     }
 
     fun trackHashingRepoSuccess() {
-        trackEvent("/hashing/repo/success")
+        trackEvent("hashing/repo/success")
     }
 
     fun trackHashingSuccess() {
-        trackEvent("/hashing/success")
+        trackEvent("hashing/success")
     }
 
     fun trackError() {
         // TODO(anatoly): Send message with exd, consider limit 150 bytes.
-        trackEvent("/error", listOf("t" to HIT_EXCEPTION))
+        trackEvent("error", listOf("t" to HIT_EXCEPTION))
```

Comment:
Normally, you don't want to change your logged events because those that are logged will stay in GA forever. So any change to log format, like renaming events, will impact funnels and their historical availability. I will let this one slip this one time, but please follow up with detailed error codes asap.

Reply:
I think it's better to use special error tracking software, e.g. https://sentry.io, because error code is not enough.

Reply:
For GA error urls probably best solution is to use exception names, agree?